packages/generators/add-generator.ts

```diff
@@ -85,6 +42,12 @@ export default class AddGenerator extends Generator {
 			configName?: string;
 			topScope?: string[];
 			item?: string;
+			merge?: {
```

Comment:
Sure!
I had a problem with this one 😅, as the merge strictly can't have merge property, it will act as a bad interface. Can you suggest a workaround?

Reply:
I think maybe we should discuss how we should handle merge (see the comment on add generator) and then resume this if necessary

Reply:
Never mind, I just read the next comment, not a problem now :)

Reply:
Solved now, Thanks!